### PR TITLE
[ra2 ch2] sync ch2 to 4 references 2.2.1 to 2.2.5

### PR DESCRIPTION
**Pull request by rgstori**
closes partially #2259 
I've added references to existing ch4 specs and put "NA" where there was none.
For the mandatory requirements that are not backed by a spec, I added issues #2705, #2706, #2707, #2708, #2709

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 1]
```diff
@@ -45,31 +46,31 @@ To ensure alignment with the infrastructure profile catalogue, the following req
...
 | [4.1.4](../../../ref_model/chapters/chapter04.md#414-internal-infrastructure-capabilities) | i.cap.015 | Indicates the memory consumed by Cloud Infrastructure on the worker nodes | 16 GB | 16GB | |
 | [4.1.4](../../../ref_model/chapters/chapter04.md#414-internal-infrastructure-capabilities) | i.cap.016 | Number of virtual cores per physical core; also known as CPU overbooking ratio that is required | 1:1 | 1:1 | [ra2.ch.004](chapter04.md#42-kubernetes-node)<br>[ra2.ch.005](chapter04.md#42-kubernetes-node)|
-| [4.1.4](../../../ref_model/chapters/chapter04.md#414-internal-infrastructure-capabilities) | i.cap.017 | QoS enablement of the connection point (vNIC or interface)| Not required | Must support | |
+| [4.1.4](../../../ref_model/chapters/chapter04.md#414-internal-infrastructure-capabilities) | i.cap.017 | QoS enablement of the connection point (vNIC or interface)| Not required | Must support | **N/A** |
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter02.md:65]: @rgstori why bold the N/A (also in other entries)?

Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter02.md:65]: I've bolded the N/As in requirements that aren't optional or not required, to signal that there **should** be something (a ch4 spec) implementing it, but not at the moment

**Review by walterkozlowski — APPROVED** [review]

**Review by gkunz — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@pgoyal01, can you re-review and approve, if you can?

**Comment by pgoyal01** [comment]
@walterkozlowski @rgstori some suggested changes that have not been commented/acted upon.

**Comment by walterkozlowski** [comment]
@rgstori, can you please try to resolve this?

**Comment by walterkozlowski** [comment]
@rgstori, this seems to be the last "proofreading" PR left.  Can you please reconcile with comments from @pgoyal01, so I could merge?  Thanks.

File: doc/ref_arch/kubernetes/chapters/chapter02.md (hunk 1 above)
Comment (karinesevilla) [reply doc/ref_arch/kubernetes/chapters/chapter02.md:65]: @rgstori The meaning of N/A and N/A bold must be added. Is it N/A for not applicable and N/A bold for not available?

Comment (rgstori) [reply doc/ref_arch/kubernetes/chapters/chapter02.md:65]: ok, have added a definition in https://github.com/cntt-n/CNTT/pull/2710/commits/a0b2b28529493b85a13aa2d7095bca51094a6d72

**Comment by rgstori** [comment]
should be solved in https://github.com/cntt-n/CNTT/pull/2710/commits/a0b2b28529493b85a13aa2d7095bca51094a6d72

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 2]
```diff
@@ -341,7 +341,7 @@ With regards to containerisation platforms, the scope of the following Architect
...
-| kcm.gen.01 | Kubernetes Cluster | General | The Architecture **must** support policy driven horizontal auto-scaling of Kubernetes Cluster. | |
+| kcm.gen.01 | Kubernetes Cluster | General | The Architecture **must** support policy driven horizontal auto-scaling of Kubernetes Cluster. | **N/A** |
 | kcm.gen.02 | Kubernetes Cluster | General | The Architecture **must** enable workload resiliency. |[ra2.k8s.004](chapter04.md#43-kubernetes)|
 | int.api.01 | API | General | The Architecture **must** leverage the Kubernetes APIs to discover and declaratively manage compute (virtual and bare metal resources), network, and storage. |For Networking: <ul><li>[ra2.ntw.001](chapter04.md#45-networking-solutions)<li>[ra2.ntw.008](chapter04.md#45-networking-solutions)<li>[ra2.app.006](chapter04.md#49-kubernetes-workloads)</ul><br>Compute/storage not yet met. |
 | int.api.02 | API | General | The Architecture **must** support the usage of a Kubernetes Application package manager using the Kubernetes API, like Helm v3. |[ra2.pkg.001](chapter04.md#48-kubernetes-application-package-manager)|
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter02.md:347]: This is a test comment

**Review by pgoyal01 — APPROVED** [review]

**Pull request merged**